source/howto/mobility/getstarted.rst

```diff
@@ -215,14 +265,15 @@ The meaning of the keys are:
   TrafficStation, EChargingPlug, Bicycle, and so on.
   
   .. note:: This key is :strong:`Case Sensitive`! You can retrieve all
-     the station types with the following call:
+     the station types with the following one-liner, which pipes the
+     call to the list of all Stations to the :command:`jq` command:
 
      .. code::
 	
-	curl -X GET "https://mobility.api.opendatahub.bz.it/v2/" -H "accept: application/json"
+	curl -X GET "https://mobility.api.opendatahub.bz.it/v2/tree" -H "accept: application/json"  |  jq '.[].id' 
```

Comment:
Would not add `jq` here, that is up to the developer... I would just talk about the API and not CLI specific commands